.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,88 @@
+name: 'Bug report'
+description: Create a report to help us improve
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for reporting an issue :pray:.
+
+        This issue tracker is for reporting bugs found in `NextUI` (https://github.com/nextui-org/nextui/).
+        If you have a question about how to achieve something and are struggling, please post a question
+        inside of either of the following places:
+          - NextUI's Discussion's tab: https://github.com/nextui-org/nextui/discussions
+          - NextUI's Discord channel: https://foambubble.github.io/join-discord/g
+          
+
+        Before submitting a new bug/issue, please check the links below to see if there is a solution or question posted there already:
+         - NextUI's Issue's tab: https://github.com/nextui-org/nextui/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc
+         - NextUI's closed issues tab: https://github.com/nextui-org/nextui/issues?q=is%3Aissue+sort%3Aupdated-desc+is%3Aclosed
+         - NextUI's Discussions tab: https://github.com/nextui-org/nextui/discussions
```

Comment:
Same Goes here (wrap links as text) as it looks way more clean :- 

> also can you please replace discord with direct invite [link](https://discord.gg/9b6yyZKmH4) 

![image](https://user-images.githubusercontent.com/47073516/156115165-b1748a8d-0b42-4d09-840d-794acaf784b8.png)

**_Expected Output:-_**

![image](https://user-images.githubusercontent.com/47073516/156115205-98b9e7b6-b94b-48be-8967-7e61f9ed5184.png)